### DISCUSSION TITLE
unable to set the quality of the recorded files from OME

### DISCUSSION THREAD

**Discussion by magnuszetterberg** [OvenMediaLabs/OvenMediaEngine#1224] [Help]
Not a bug, but rather a question.
I cant find any settings in the documentation regarding the quality of the recorded streams. Is there such a setting, but simply not documenteted for OME, so that I for example can set the bitrate on the .mp4 or .ts file which I want to record the stream to?

A simple "yes, there are settings for this, and they are the following..." or "...no, its not possible to set the quality of the recorded stream in OME".

Thank you very much!!

**Comment by naanlizard** [comment]
I would assume the stream is recorded at the same quality and settings of the input.

If you want to re-encode for recording I think you can just set up a new encoder for incoming streams, and then record that variant

**Reply by magnuszetterberg** [reply]
Hello!

Since it not possible to show a live stream , and then show the recorded equvalient of the live stream(due to it being live) I cant prove anything, more then actually look at the impression of the quality of the live stream, and them compare it to the recorded stream, which to me show degradation regarding macro-blocks which seems to be due to a lower bitrate compared to the live feed....

**Comment by getroot** [comment]
You can record the tracks you want with "variantNames" in the body of the Start Recording API request.

https://airensoft.gitbook.io/ovenmediaengine/rest-api/v1/virtualhost/application/recording#start-recording

**Reply by magnuszetterberg** [reply]
not any point of these links show any settings regarding the bitrate or a similar setting when saving a livestream being pushed though the OME server....  or am I incorrect in this statement?

**Reply by getroot** [reply]
I don't understand what exactly you want. What you want is not to record the input stream as it is, but to record with the quality converted, right?

If so, re-encode the input video as follows.
https://airensoft.gitbook.io/ovenmediaengine/transcoding

Then write the names of the tracks you want to record in variantNames in the body of the Start Recording API request.
